modules/site-to-site-vpn/main.tf

```diff
@@ -15,7 +15,7 @@ locals {
   ) : null
 }
 
-module "site_to_site_vpn" {
```

Comment:
this'll cause a destroy and replace for anyone with this component currently deployed. Although this isnt particular common, we should at least include a changelog to call this out and warn anyone vendoring

Reply:
This should actually be a fix, the outputs of this component refer to `module.vpn_connection` - which is this module, or meant to be. So it was not deployable without renaming it.